docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,286 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Contributing to SpECTRE {#contributing_to_spectre}
+
+The following is a set of guidelines for contributing to SpECTRE,
+which is hosted in the [Simulating eXtreme Spacetimes
+Organization](https://github.com/sxs-collabortion) on GitHub.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
```

Comment:
These anchors don't work. Not sure how to fix

Reply:
I had to use html anchors; I get warnings with doxygen labels

Reply:
I've figured out how to use the standard doxygen labels